include/ts/apidefs.h.in

```diff
@@ -902,8 +902,19 @@ typedef enum {
   TS_THREAD_POOL_UDP
 } TSThreadPool;
 
+/* TSHRTime stuff, this is a candidate for deprecation in v10.0.0 */
 typedef int64_t TSHRTime;
 
+#define TS_HRTIME_SECOND  (1000 * TS_HRTIME_MSECOND)
+#define TS_HRTIME_MSECOND (1000 * TS_HRTIME_USECOND)
+#define TS_HRTIME_USECOND (1000 * TS_HRTIME_NSECOND)
+#define TS_HRTIME_NSECOND (1LL)
+
+#define TS_HRTIME_SECONDS(_x)  ((_x)*TS_HRTIME_SECOND)
```

Comment:
Why not `constexpr TSHRTime TS_HRTIME_SECONDS(TSHRTIME nsec) { return nsec * TS_HRTIME_SECOND); }` ?